documentation/src/main/asciidoc/query.asciidoc

```diff
@@ -289,6 +289,74 @@ Query luceneQuery = mythQB.keyword()
 In the previous example, only field name is boosted to 5.
 
 
+===== Plain text queries
+
+Plain text queries use the Lucene SimpleQueryParser. The query text is parsed and it exposes more
+search features to the end user such as boolean operators and phrase queries.
+
+====
+[source, JAVA]
+----
+Query luceneQuery = mythQB
+    .text()
+    .onField("history")
+    .matching("(storm OR hurricane) AND \"falling branches\"")
+    .createQuery();
+----
+====
+
+By default, a plain text query uses the OR operator if the operator is not explicitly defined.
+If you prefer using the AND operator as default, you can add the `defaultOperatorIsAnd` flag.
+The following query returns results containing storm AND tree.
+
+====
+[source, JAVA]
+----
+Query luceneQuery = mythQB
+    .text()
+    .onField("history")
+    .defaultOperatorIsAnd()
```

Comment:
This is a bit clumsy.

So...:

 1. `defaultOperator(Operator.AND)` isn't much better
 2. `or()`/`and()` lack some context
 3. I guess `.conjunctive`/`.disjunctive` are a tad too obscure

If nobody finds a better option, I think I'd say we go for 1... Even though it's far from ideal.